### PR TITLE
Reader post card: show post likers in gravatar caterpillar

### PR DESCRIPTION
**Pull request by bluefuton**
Based on @jancavan's designs in p5PDj3-4fQ, show post likers on the Reader post card and at the end of full posts:

likes0061

### Todo
- [ ] Pass total count to caterpillar (from `countPostLikes`)
- [ ] Move likes popover to caterpillar from like button
- [ ] Fix like count for smaller screens (where fewer avatars are shown)

### CODE REVIEW

**Comment by bluefuton** [comment]
@jancavan this is far from finished, but sharing for early feedback :)

I've left the gravatars in the same spot for post cards and full post for the moment, as they share a component (`reader-post-actions`). Do you still think we should drop the 'Visit' button and shuffle the caterpillar to the left in the stream?

**Comment by jancavan** [comment]
Thanks @bluefuton! 

Yes, there should be labels and gravs should be smaller like the mockup. I think we should still keep the Like count on the Like button itself, that's just an mistake on the mockup.

**Comment by bluefuton** [comment]
I've gone with 12px gravs and added the like and comment count on the left hand side 👍 

Does feel like we're duplicating the key information though:

screen shot 2018-07-19 at 17 54 06

File: client/blocks/reader-post-actions/index.jsx [hunk 1]
```diff
@@ -44,24 +48,40 @@ const ReaderPostActions = props => {
...
-					</ReaderVisitLink>
+					<PostLikesCaterpillar
+						blogId={ +post.site_ID }
+						postId={ +post.ID }
+						gravatarSize={ 12 }
+						className="reader-post-actions__post-likes-caterpillar"
+					/>
+					{ commentCount > 0 && (
+						<div className="reader-post-actions__item-text">
+							{ translate( '%(count)d comment', '%(count)d comments', {
```
Comment (a8ci18n) [inline client/blocks/reader-post-actions/index.jsx]: Hi! I've found a possible matching string that has already been translated **41** times:
`translate( '%d comment', '%d comments' )` *ES Score: 14*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/reader/post-likers-to-post-card)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by jancavan** [comment]
I think we can remove (2+) like:

screenshot 2018-07-19 09 14 47

I think maybe 3 works better as default. 5 looks too crowded now that I see it.

In the original post, the intention was to combine gravs of both likes and comments, at least in the card. Is this doable?

> Add a separate section for Comment/Like count and display the gravatars of those who commented and liked the post

> Does feel like we're duplicating the key information though:

I think it's fine since they're far enough apart.

**Comment by bluefuton** [comment]
I've reduced to 3 gravs and turned off the count 👍 

> In the original post, the intention was to combine gravs of both likes and comments, at least in the card. Is this doable?

I'm sure we can figure that out. Do we want to signify a like or comment in any way, like @shaunandrews original party people concept? https://github.com/Automattic/wp-calypso/issues/745

**Comment by jancavan** [comment]
> Do we want to signify a like or comment in any way

That's actually a good idea. For now though, they're a bit too small to have something superimposed on them so we can keep it as is and re-explore later.

**Comment by bluefuton** [comment]
I think this was a useful exploration. I'll close it out for the moment until we loop back to work on this again.

**Pull request closed without merging**